var/spack/repos/builtin/packages/py-jupyter-server/package.py

```diff
@@ -14,43 +14,15 @@ class PyJupyterServer(PythonPackage):
     homepage = "https://github.com/jupyter-server/jupyter_server"
     pypi     = "jupyter_server/jupyter_server-1.9.0.tar.gz"
 
-    import_modules = ['jupyter_server', 'jupyter_server.services',
```

Comment:
It looks like @iarspider added these in #27153. @iarspider can you confirm that the modules detected by default are fine?

Reply:
Yes, looks like all the modules are importable now.

Reply:
I just checked and the modules automatically found by spack are the same:
```
==> [2022-02-23-10:29:10.977166] Detected the following modules: ['jupyter_server', 'jupyter_server.extension', 
'jupyter_server.tests', 'jupyter_server.tests.extension', 'jupyter_server.tests.extension.mockextensions', 
'jupyter_server.tests.unix_sockets', 'jupyter_server.tests.auth', 'jupyter_server.tests.services', 
'jupyter_server.tests.services.contents', 'jupyter_server.tests.services.kernels', 'jupyter_server.tests.services.api', 
'jupyter_server.tests.services.config', 'jupyter_server.tests.services.kernelspecs', 'jupyter_server.tests.services.sessions', 
'jupyter_server.tests.services.nbconvert', 'jupyter_server.tests.nbconvert', 'jupyter_server.gateway', 
'jupyter_server.prometheus', 'jupyter_server.auth', 'jupyter_server.kernelspecs', 'jupyter_server.services', 
'jupyter_server.services.contents', 'jupyter_server.services.kernels', 'jupyter_server.services.api', 
'jupyter_server.services.config', 'jupyter_server.services.kernelspecs', 'jupyter_server.services.security', 
'jupyter_server.services.sessions', 'jupyter_server.services.nbconvert', 'jupyter_server.view', 'jupyter_server.base', 
'jupyter_server.terminal', 'jupyter_server.i18n', 'jupyter_server.nbconvert', 'jupyter_server.files']
```